### PR TITLE
Integrate buildtools: buildifier and unused_deps

### PR DESCRIPTION
**Pull request by vmax**

### CODE REVIEW

File: commands.txt [hunk 1]
```diff
@@ -24,3 +24,16 @@ bazel run @bazel_deps//:parse -- add-dep --deps `pwd`/dependencies.yaml --lang j
 # Usually there is already a folder 'third_party' but if there isn't, comment out loading of //third_party/maven:workspace.bzl in WORKSPACE
 bazel run @bazel_deps//:parse -- generate -r `pwd` -s third_party/maven/workspace.bzl -d dependencies.yaml
 
+# Buildtools (https://github.com/bazelbuild/buildtools)
+
+# Fix formatting of BUILD file with `buildifier`
```
Comment (oferb) [inline commands.txt]: How about this:
```
# Fix formatting of BUILD file with `buildifier`
# (https://github.com/bazelbuild/buildtools/tree/master/buildifier)
# Note that all paths should be absolute
bazel run //tools:buildifier -- -mode=fix `pwd`/…

# Find unused dependencies with `unused_deps`
# (https://github.com/bazelbuild/buildtools/blob/master/unused_deps/)
bazel run //tools:unused_deps -- <BUILD target, e.g //folder:target>
```

File: tools/buildtools_wrappers/unused_deps.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    UNUSED_DEPS_BINARY=$(pwd)/external/unused_deps_osx/file/unused_deps.osx
+elif [ "$platform" == "Linux" ]; then
+    UNUSED_DEPS_BINARY=$(pwd)/external/unused_deps/file/unused_deps
+else
+    echo "unused_deps does not have binary for $platform"
+    exit 1
+fi
+
+cd $BUILD_WORKSPACE_DIRECTORY
+$UNUSED_DEPS_BINARY $*
```
Comment (oferb) [inline tools/buildtools_wrappers/unused_deps.sh]: After you run this, does your current folder change?

File: tools/buildtools_wrappers/buildifier.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+# Wrapper script for running buildifier
+
+platform=$(uname)
+
+if [ "$platform" == "Darwin" ]; then
+    BUILDIFIER_BINARY=$(pwd)/external/buildifier_osx/file/buildifier.osx
+elif [ "$platform" == "Linux" ]; then
+    BUILDIFIER_BINARY=$(pwd)/external/buildifier/file/buildifier
+else
+    echo "Buildifier does not have binary for $platform"
```
Comment (oferb) [inline tools/buildtools_wrappers/buildifier.sh]: s/have binary/have a binary

File: tools/buildtools_wrappers/unused_deps.sh (hunk 2 above)
Comment (vmax) [reply tools/buildtools_wrappers/unused_deps.sh]: @oferb it doesn't, `bazel run` runs script in a separate sub-shell

**Pull request merged**